### PR TITLE
Make channel active when joining from channel list on small screens

### LINKED ISSUE #719: Channel List Improvement
48370985_1231934476947094_6593283705739935744_n

Return to Channel from the Channel List is not not very intuitive on mobiles 

[ 0 ] does not indicate to user as a return button

### PR DESCRIPTION
**Pull request by ItsOnlyBinary**
closes #719

### CODE REVIEW

File: src/components/ChannelList.vue [hunk 1]
```diff
@@ -155,6 +155,9 @@ export default {
         joinChannel(channelName) {
             this.$state.addBuffer(this.network.id, channelName);
             this.network.ircClient.join(channelName);
+            if (this.$state.ui.app_width <= 769) {
```
Comment (ItsOnlyBinary) [inline src/components/ChannelList.vue]: I've used this in more than one place, not sure if it should be made into a method in state or somewhere more appropriate

**Comment by ItsOnlyBinary** [comment]
This will now also switch the active buffer when join is clicked on an already joined channel

**Comment by finalhit** [comment]
@ItsOnlyBinary
> on large screens it will open the channel on first click and switch on second click (this is to allow joining multiple channels without it switching away from the channel list)

Second click does not switch to the new channel, just tried.

Anyway this is very strange implementation. I am not aware of IRC clients opening channels in the background. 

I think it should always make channel active when joining from channel list, not on small screens only.

**Comment by ItsOnlyBinary** [comment]
test it [here](https://builds.kiwiirc.com/kiwiirc/pull-755/) as this is a pull request so has not been merged with master

**Comment by finalhit** [comment]
I tested and it looks pretty unintuitive.

It even encourages spammers. Someone could easily join hundreds of channels in no time.

**Pull request closed without merging**